LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Alexandru Somai and Jenkins contributors
```

Comment:
I don't know exactly how this should be done.
Should I also mention explicitly other contributors to this project? Ikeda Yasuyuki and Alan Harder?